mlir/lib/Conversion/AMDGPUToROCDL/AMDGPUToROCDL.cpp

```diff
@@ -385,6 +385,7 @@ static void wmmaPushInputOperand(ConversionPatternRewriter &rewriter,
                                  Location loc,
                                  const TypeConverter *typeConverter,
                                  bool isUnsigned, Value llvmInput,
+                                 Value mlirInput,
```

Comment:
This prefix is surprising to me, since this is mlir code and any input would be an mlir input...?

Reply:
This is because LLVM does not have an fp8 data type. fp8 gets converted to int8. So I need to have both the value before conversion (`mlirInput`) and the value after conversion(`llvmInput`)

Reply:
I added a comment to clarify this, PTAL